highfidelity/hifi
# Set active button property isActive before Home menu redraw. Makes su…

…re redraw will read proper value of isActive property

Test plan:
- Press Create button
- Press Home aka (X) button 
- repeat, during high system load
- make sure Create button did not stays in active mode
